NORTRIP_main_run.f90

```diff
@@ -87,54 +98,81 @@ subroutine NORTRIP_main_run
             write(unit_logfile,'(A)')'Starting time loop (NORTRIP_main_run)'
         endif
         
-        do ti=min_time,max_time
-      
-            !Print the day date. Not active
-            if (date_data(hour_index,ti).eq.1) then
-                !write(unit_logfile,'(I5,I3,I3)') date_data(year_index,ti),date_data(month_index,ti),date_data(day_index,ti)
-            endif
-              
-            !Use activity rules to determine salting, sanding and cleaning activities
-            call NORTRIP_set_activity_data
-        
-            !Main track loop
-            !----------------------------------------------------------------------
-            do tr=1,num_track
-         
-                !Calculate road surface conditions
-                call NORTRIP_surface_moisture_submodel
-                
-                !For debugging purposes only
-                if (show_time_moisture) then
-                    write(unit_logfile,'(a24,a2,2i8,f8.2,f8.1,f8.2,f8.2,f8.2,f8.2,f8.2,f8.2)') trim(date_str(3,ti)),': ',ro,ti &
-                    ,meteo_data(T_a_index,ti,ro),meteo_data(RH_index,ti,ro),road_meteo_data(T_s_index,ti,tr,ro),road_meteo_data(T_sub_index,ti,tr,ro),road_meteo_data(T_s_dewpoint_index,ti,tr,ro) &
-                    ,g_road_data(water_index,ti,tr,ro),g_road_data(snow_index,ti,tr,ro),g_road_data(ice_index,ti,tr,ro)
-                endif
-                
-                !Calculate road emissions and dust loading
-                call NORTRIP_dust_emission_submodel
-                if (show_time_dust) then
-                    write(unit_logfile,'(a24,a2,2i8,f8.2,f8.1,f8.2,f8.2,f8.2,f8.2,f8.2,f8.2)') trim(date_str(3,ti)),': ',ro,ti &
-                    ,road_meteo_data(H_index,ti,tr,ro),road_meteo_data(L_index,ti,tr,ro),road_meteo_data(G_index,ti,tr,ro),road_meteo_data(G_sub_index,ti,tr,ro),road_meteo_data(evap_index,ti,tr,ro) &
-                    ,road_meteo_data(RH_s_index,ti,tr,ro),M_road_bin_data(road_index,pm_200,ti_bin,tr,ro_bin),M_road_bin_data(salt_index(1),pm_all,ti_bin,tr,ro_bin)
+        do tf=min_time,max_time
```

Comment:
For the time loop: Now uses tf as counter for the outer loop, and ti for the inner loop like in the matlab version.